boltdesignsystem/bolt
# DS-152 Page Header Refactor

## Jira

https://pegadigitalit.atlassian.net/browse/DS-152

## Summary

Adds Page Header component.

## Details

1. Page header component has been properly built out.
2. It is no longer just a PL demo component.
3. Accessibility and ux enhancements are implemented.
4. Will test out Drupal implementation, starting with .com. (@remydenton)

## How to test

Run the branch locally and check all docs and demos under Page Header in Pattern Lab.
